src/index.html

```diff
@@ -7,8 +7,52 @@
     <meta http-equiv="X-UA-Compatible" content="ie=edge">
     <title>Moyo header</title>
     <link rel="stylesheet" href="./style.css">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link 
+      rel="preconnect" 
+      href="https://fonts.gstatic.com"
+      crossorigin="what"
+    >
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@500&display=swap" rel="stylesheet">
   </head>
   <body>
-    <h1>Moyo header</h1>
+    <header class="header1">
+      <div class="moicon">
+        <a href="#home" class="iconmy__link">
+          <img
+            src="./images/logo.png"
+            alt="MOYO"
+            class="iconmy__img"
+          >
+        </a>
+      </div>
+      <nav class="navblock ">
+        <ul class="nav__list">
+          <li class="nav__item">
+            <a 
+              href="#"
+              class="nav__link is-active"
+            >
+              Apple
+            </a>
+          </li>
+          <li class="nav__item"><a href="#" class="nav__link">Samsung</a></li>
+          <li class="nav__item"><a href="#" class="nav__link">Смартфоны и телефоны</a></li>
```

Comment:
```suggestion
          <li class="nav__item">
            <a href="#" class="nav__link">
              Samsung
            </a>
          </li>
          <li class="nav__item">
            <a href="#" class="nav__link">
              Смартфоны и телефоны
            </a>
          </li>
```

and check the same below